node-webpack/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack",
+    "build": "webpack && cp -R ./node_modules ./dist/",
```

Comment:
I think this is not necessary as the dist files aren't being copied anywhere in this example.

Reply:
Actually index.js is created in the dist folder
See:
https://github.com/Rookout/deployment-examples/blob/41d58c5d6eb0e4634f6495ad1bbda1e8fdfd8616/node-webpack/webpack.config.js#L9
And eventually we run it from dist, thus we need the node_modules there:
https://github.com/Rookout/deployment-examples/blob/41d58c5d6eb0e4634f6495ad1bbda1e8fdfd8616/node-webpack/package.json#L9

Reply:
Running `node dist/index.js` works on my machine without copying the node_modules.

According to: https://docs.npmjs.com/cli/v7/configuring-npm/folders#more-information
> Starting at the $PWD, npm will walk up the folder tree checking for a folder that contains either a package.json file, or a node_modules folder. If such a thing is found, then that is treated as the effective "current directory" for the purpose of running npm commands.